src/Illuminate/Foundation/Testing/TestResponse.php

```diff
@@ -540,6 +540,27 @@ public function assertJsonMissingExact(array $data)
         );
     }
 
+    /**
+     * TestResponse::assertJsonFragment and TestResponse::assertJsonMissing search for
+     * JSON encoded strings in the Response. This function returns the set of strings to
+     * be searched for in the response, explicitly adding all of the possible characters
+     * which could follow $value in the response.
+     *
+     * @param $key
+     * @param $value
+     * @return array
+     */
+    protected function jsonSearchStrings($key, $value)
+    {
+        $needle = substr(json_encode([$key => $value]), 1, -1);
+
+        return [
+            $needle.']',
+            $needle.'}',
+            $needle.',',
+        ];
```

Comment:
This won't work for strings?

Reply:
See https://github.com/Juddling/framework/blob/85dbe29d646e790bbdd3f110452e49b07222eec6/tests/Foundation/FoundationTestResponseTest.php#L164 for existing tests which cover searching for strings.